server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/DownloadGet.feature

```diff
@@ -80,4 +81,22 @@ Feature: Download GET
     And a connected user "username1@domain.tld"
     And "username1@domain.tld" has a mailbox "INBOX"
     When "username1@domain.tld" downloads "1"
-    Then the user should receive a not found response
\ No newline at end of file
+    Then the user should receive a not found response
+
+  Scenario: User cannot download attachment of another user when shared mailbox
```

Comment:
is it possible to fixup that: s/cannot/can/

Reply:
Already done

Reply:
not as a fixup, the first commit 7a793eebe30e6380f35dba1b7b97907c36c4ddaa is really wrong alone